### PR TITLE
Toronto 

### PR DESCRIPTION
**Pull request by jwager0408**
A general set of notes concerning Toronto as an example

- [ ] add places to eat
- [ ] add other things

### CODE REVIEW

File: Toronto [hunk 1]
```diff
@@ -0,0 +1,3 @@
+Toronto is a great place.
+Hit the CN Tower
```
Comment (jwager0408) [inline Toronto]: @kkarpala do you think this works

**Comment by jwager0408** [comment]
Test comment.

File: Toronto (hunk 1 above)
Comment (nathyga) [inline Toronto]: I think you should add the aquarium as well.

Comment (kkarpala) [inline Toronto]: Kensington Market, The Beaches and Toronto Islands are a must

**Comment by kevinmalhotra** [comment]
The food festivals are excellent.  Did you enjoy Taste of the Danforth?

**Pull request merged**